A2rchi/chains/base.py

```diff
@@ -88,19 +91,27 @@ def from_llm(
         combine_docs_chain_kwargs = combine_docs_chain_kwargs or {}
         _prompt = QA_PROMPT
         document_variable_name = "context"
+
+        logfile = data_path+"chain_input_output.log"
```

Comment:
This should probably be done using `os.path.join()`. This way it doesn't matter if `data_path` has a "/" on the end of it or not.

Reply:
Also, may want to consider "chain_input_output.log" to be a configurable parameter. But I'll leave this to you.

For context, I've been meaning to change "conversations.json" to a configurable parameter.

If you do put it into the config, make sure to write a comment on what it is and any specifications (if it has to end in .log, etc.)

Not required though, just a thought